ui/arduino2/store.js

```diff
@@ -671,11 +671,17 @@ async function selectDiskFolder() {
 }
 
 async function getDiskFiles(path) {
-  const files = await disk.ilistFiles(path)
-  return files.map(f => ({
+  let files = await disk.ilistFiles(path)
+  files = files.map(f => ({
     fileName: f.path,
     type: f.type
   }))
+  files = files.sort(filesSortAlphabetically)
+  return files
+}
+
+function filesSortAlphabetically(entryA, entryB) {
```

Comment:
I would like to propose the function name to be `sortFilesAlphabetically` to more or less follow the verb-noun names of other methods.